### PR TITLE
First pass at disabling force-fetch in SSR contexts

### PR DESCRIPTION
**Pull request by tmeasday**
This is just a start in the direction of https://github.com/apollostack/apollo-client/issues/298

If we think it's the right approach (modifying query options at the `ApolloClient` layer), then I'll add tests and turn it into a proper PR.

TODO:

- [ ] Update CHANGELOG.md with your change
- [ ] Make sure all of the significant new logic is covered by tests
- [ ] Rebase your changes on master so that they can be merged easily
- [ ] Make sure all tests and linter rules pass

### CODE REVIEW

**Comment by stubailo** [comment]
This looks pretty good to me!

**Comment by tmeasday** [comment]
Added tests, please take a look.

File: test/client.ts [hunk 1]
```diff
@@ -879,4 +880,103 @@ describe('client', () => {
         });
     });
   });
+
+  describe.only('forceFetch', () => {
```
Comment (stubailo) [inline test/client.ts]: What does describe.only do?

**Comment by stubailo** [comment]
Looks good to me!

File: test/client.ts (hunk 1 above)
Comment (tmeasday) [inline test/client.ts]: Whoops, just runs those tests.

**Comment by tmeasday** [comment]
Hmm, and somehow I broke other tests, investigating.

**Comment by tmeasday** [comment]
Ok, this is working now.

**Pull request merged**